src/app/app.cpp

```diff
@@ -699,6 +699,26 @@ bool CApplication::Create()
     return true;
 }
 
+void CApplication::Reload()
+{
+    m_sound->Create();
```

Comment:
You are calling Create() on reload, this function is supposed to be only called once. You are probably reallocating resources without freeing old ones.